2023-08-03 13:47:06,778 - ERROR: An error occurred: datasets should not be an empty iterable
Traceback (most recent call last):
  File "dil_flair_id.py", line 219, in main
    trainset = ConcatDataset(train_datasets)
  File "/d/maboum/.local/lib/python3.8/site-packages/torch/utils/data/dataset.py", line 222, in __init__
    assert len(self.datasets) > 0, 'datasets should not be an empty iterable'  # type: ignore[arg-type]
AssertionError: datasets should not be an empty iterable

